HW4

```diff
@@ -0,0 +1,35 @@
+#HW4_1
+def solution(A):
```

Comment:
easy way 

    for i in range(1, len(l) + 2):
        if i not in l:
            return i